src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -155,23 +157,33 @@ public static Auth0UserProfile getUserById(String id) {
         try {
             user = mapper.readValue(response, Auth0UserProfile.class);
         } catch (IOException e) {
+            LOG.error("Unable to parse user profile response from Auth0! Response: {}", response);
             e.printStackTrace();
         }
         return user;
     }
 
     /**
-     * Get users subscribed to a given target ID.
+     * Sets uri builder scheme port and host according to whether a test environment is in effect
      */
-    public static String getUsersBySubscription(String subscriptionType, String target) {
-        return getAuth0Users("app_metadata.datatools.subscriptions.type:" + subscriptionType + " AND app_metadata.datatools.subscriptions.target:" + target);
+    private static void setSchemePortAndHost(URIBuilder builder) {
```

Comment:
This may be relatively minor, but I don't love the idea of passing a builder into this function, having the URL set, and returning void. I think a more straightforward approach might be to just have a method called `constructAuthURIBuilder` that constructs the URIBuilder, sets the URL stuff, and returns it. For example, this:

```
URIBuilder builder = new URIBuilder();
setSchemePortAndHost(builder);
```

would become: 

```
URIBuilder builder = constructAuthURIBuilder();
```

Reply:
done.